webapp/channels/src/packages/mattermost-redux/src/reducers/entities/channel_categories.ts

```diff
@@ -92,6 +95,44 @@ export function byId(state: IDMappedObjects<ChannelCategory> = {}, action: Gener
 
         return changed ? nextState : state;
     }
+    case ChannelTypes.RECEIVED_CHANNEL: {
+        const receivedChannel: Channel = toClientChannel(action.data);
```

Comment:
0/5, but you don't actually need to use `toClientChannel` here since it's just stripping off some extra fields that we don't want to store with the rest of the channel object in redux

Reply:
Done.